README.md

```diff
@@ -16,6 +16,8 @@ Recommended practices
 - Should be [incremental](#incremental-queries)
 - Should filter input tables on partition and clustering columns
 - Should use `_` prefix in generated column names not meant for output
+- Should not use `DATETIME` type, due to incompatiblity with
+  [spark-bigquery-connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector/issues/5)
```

Comment:
This is different from the storage API problem where we can't choose partitions for tables partitioned on datetime? We can't access datetime types at all in spark due to this issue?

Reply:
yes to both.

Also, to be clear, it's possible to read `DATETIME` values in the *API*, but the *connector* throws an error because it doesn't have a timezone for `DATETIME` values -_-